itwin-platform-access/imodels-access-frontend/package.json

```diff
@@ -2,7 +2,21 @@
   "name": "@itwin/imodels-access-frontend",
   "version": "0.1.0",
   "description": "Interoperability package between iModels API and iTwin platform for frontend.",
```

Comment:
iTwin.js library instead of iTwin platform.

Reply:
Fixed: https://github.com/iTwin/imodels-clients/blob/8d1ba8536b41d0b6bb1aac2118b9ac456c6a0324/itwin-platform-access/imodels-access-frontend/package.json#L4